scripts/snyk-test.js

```diff
@@ -0,0 +1,89 @@
+const childProcess = require('child_process');
+const path = require('path');
+const { promises: fs } = require('fs');
+const os = require('os');
+const { glob } = require('glob');
+
+async function spawn(cmd, args, opts) {
+  return new Promise((resolve, reject) => {
+    const proc = childProcess.spawn(cmd, args, opts);
+    proc.on('error', (err) => reject(err));
+    proc.on('close', (code) => resolve(code));
+  });
+}
```

Comment:
```suggestion
const { promisify } = require('util');
const spawn = promisify(childProcess.execFile);
```

(from the Node.js docs:)

> If this method is invoked as its [util.promisify()](https://nodejs.org/api/util.html#utilpromisifyoriginal)ed version, it returns a Promise for an Object with stdout and stderr properties. The returned ChildProcess instance is attached to the Promise as a child property. In case of an error (including any error resulting in an exit code other than 0), a rejected promise is returned, with the same error object given in the callback, but with two additional properties stdout and stderr.

> ```js
> const util = require('node:util');
> const execFile = util.promisify(require('node:child_process').execFile);
> async function getVersion() {
>   const { stdout } = await execFile('node', ['--version']);
>   console.log(stdout);
> }
> getVersion();
> ```